packages/styling/components/progress-indicator.scss

```diff
@@ -0,0 +1,62 @@
+@import "../variables";
+
+$progress-bar-bg: $grasgroen;
```

Comment:
dit hele rijtje variabelen wordt niet in deze file gebruikt

Reply:
aangepast